include/chemfiles/capi/atom.h

```diff
@@ -188,7 +188,7 @@ CHFL_EXPORT chfl_status chfl_atom_set_property(
 
 /// Get a property with the given `name` in this `atom`.
 ///
-/// This function returns `NULL` is no property exist with the given name.
+/// This function returns `NULL` if no property exists with the given name.
```

Comment:
thanks! I am not a native speaker, so there might be more of them hiding around

Reply:
Not a problem, I'll fix 'em as I see 'em.